lms/djangoapps/certificates/tests/test_api.py

```diff
@@ -49,13 +51,13 @@
     get_certificate_invalidation_entry,
     get_certificate_url,
     is_certificate_invalidated,
-    is_on_allowlist,
+    is_on_allowlist, remove_allowlist_entry,
```

Comment:
Just curious: do you know if edX has a preference for putting all imports on their own line?

Reply:
I think it the preference is one-per-line. I didn't pay too much attention to what `isort` did after running it. I'll update this before I merge it.